src/assets/translations/en/main.json

```diff
@@ -1919,5 +1920,23 @@
         "cta": "Create one"
       }
     }
+  },
+  "nft": {
+    "floorPrice": "Floor Price",
+    "lastPrice": "Last Price",
+    "rarity": "Rarity",
+    "overview": "Overview",
+    "properties": "Properties",
+    "collection": "Collection",
+    "tokenId": "Token ID",
+    "address": "Address",
```

Comment:
As discussed yesterday, do we want to make this "Contract Address" instead? This seems to imply that this is the "unique address" of an NFT, which doesn't exist